packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -370,11 +370,11 @@ export const Popup: React.FC<PopupProps> &
   };
 
   const shouldBlurClose = e => {
```

Comment:
```suggestion
  const shouldBlurClose = (e: TYPE): boolean => {
```

Let's improve typings